eng/tools/TypeSpecValidation/src/rules/git-diff.ts

```diff
@@ -8,8 +8,8 @@ export class GitDiffRule implements Rule {
   readonly name = "GitDiff";
   readonly description = "Checks if previous rules resulted in a git diff";
 
-  async execute(): Promise<RuleResult> {
-    const git = simpleGit();
+  async execute(folder: string): Promise<RuleResult> {
+    const git = simpleGit(folder);
```

Comment:
Initialize `simpleGit` using the spec `folder` as `baseDir`.  This makes the command independent of the current directory used to run `tsv`, as it should be.